### PR TITLE
Update security vulnerabilities list for 20.0.0.10

### PR DESCRIPTION
**Pull request by clarkek123**

### CODE REVIEW

File: modules/ROOT/pages/security-vulnerabilities.adoc [hunk 1]
```diff
@@ -18,6 +18,13 @@ The following table lists the CVEs that affect Open Liberty, ordered by the rele
 |===
 |CVE |CVSS Score |Vulnerability Assessment |Versions Affected |Version Fixed |Notes
 
+|http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-4590[CVE-2020-4590]
+|5.3
+|Denial of service
+|19.0.0.5 - 20.0.0.9
+|20.0.0.10
+|Affects the feature:oauth-2.0[], feature:openidConnectServer-1.0[] features
```
Comment (Charlotte-Holt) [inline modules/ROOT/pages/security-vulnerabilities.adoc]: Could you update

>Affects the feature:oauth-2.0[], feature:openidConnectServer-1.0[] features

To

>Affects the feature:oauth-2.0[] and feature:openidConnectServer-1.0[] features

Comment (clarkek123) [reply modules/ROOT/pages/security-vulnerabilities.adoc]: Yes, what is the best way for me to make the changes?

Comment (clarkek123) [reply modules/ROOT/pages/security-vulnerabilities.adoc]: I think I updated the commit, do you see the change?

Comment (Charlotte-Holt) [reply modules/ROOT/pages/security-vulnerabilities.adoc]: You can just make the change in your `clarkek123-UpdateSecVulnerList200010` branch and commit and push them. Then they should be reflected in this PR.

**Review by Charlotte-Holt — APPROVED** [review]

**Pull request merged**